src/components/ContextBreadcrumbs.tsx

```diff
@@ -213,7 +216,7 @@ const ContextBreadcrumbs = ({
           marginLeft: 'calc(1.3em - 14.5px)',
           marginTop: '0.533em',
           minHeight: '1em',
-          visibility: hidden ? 'hidden' : undefined,
+          visibility: hidden || cursorOverlay ? 'hidden' : undefined,
```

Comment:
Perhaps you can reuse `hidden` and avoid increasing the surface area of the API?